### PR TITLE
Make sb?paste work in other channels and removes code block syntax

### PR DESCRIPTION
**Pull request by hanss314**
What's the use of sb?paste if you're just going to remove spam from a spam channel?
I was going to have it incude syntax highlighting, but the 'syntax' field only takes programming language names so yeah.

### CODE REVIEW

File: cogs/internet.py [hunk 1]
```diff
@@ -57,6 +57,12 @@ def __init__(self, bot):
             title = "{0.display_name}#{0.discriminator}".format(ctx.author)
         else:
             title = "{0.display_name}#{0.discriminator} in #{1}".format(ctx.author, ctx.channel.name)
+        
+        language = None
+        if data.startswith('```') and data.endswith('```'):
```
Comment (slice) [inline cogs/internet.py]: you can use a converter for this

File: cogs/util/bot.py [hunk 2]
```diff
@@ -76,13 +76,17 @@ def _check_table_exists(self, tablename):
         allowed = channel_ids.get('allowed_channels', None)
         blocked = channel_ids.get('blocked_channels', [])
 
+        # WTF is this?!?!?! This is horrible code!
+        # Not that I'm helping though.
+        # -hanss314
+        
         if allowed is not None:
             if channel.id not in allowed:
```
Comment (slice) [inline cogs/util/bot.py]: can't you use checks for this?

Comment (hanss314) [reply cogs/util/bot.py]: That would mean a bit of refactoring which I was too lazy to do.

File: cogs/internet.py (hunk 1 above)
Comment (hanss314) [reply cogs/internet.py]: Good point

**Comment by Bottersnike** [comment]
Can we make sb?g global as well while you're at it?

**Comment by Bottersnike** [comment]
image Problem.

**Comment by Bottersnike** [comment]
image
A new hope

**Comment by hanss314** [comment]
@slice  Added the requested changes

**Comment by qixils** [comment]
awaiting input
image

**Comment by hanss314** [comment]
Changes to server got approved, can we merge now?

**Comment by Bottersnike** [comment]
Rebased

**Pull request merged**